### DISCUSSION TITLE
I need help creating a custom link

### DISCUSSION THREAD

**Discussion by IshaanShettigar** [clientIO/joint#2209] [Q&A] [answered]
### Introduction

Might be a stupid question but I need to create a custom link that has a central green line and two black lines on either side of the green line as depicted in the picture.

![image](https://github.com/clientIO/joint/assets/77607172/c80ed951-93ae-43a4-a950-8a286a40e643)

The green line need NOT end a few pixels before the black line. 

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment, marked as answer]
You have to use the trick with multiple paths laid on top of each other.

Here's an example: https://jsfiddle.net/kumilingus/96psr5nd/

**Reply by IshaanShettigar** [reply]
Thank you so much, the support you guys provide is just amazing!
